core/src/main/java/org/ironjacamar/core/workmanager/WorkManagerCoordinator.java

```diff
@@ -416,6 +416,14 @@ public synchronized WorkManager createWorkManager(String id, String name)
             if (dwm.getTransport() != null)
             {
                dwm.getTransport().register(new Address(wm.getId(), wm.getName(), dwm.getTransport().getId()));
+               try
+               {
+                  ((DistributedWorkManager) wm).getTransport().startup();
+               }
+               catch (Throwable t)
```

Comment:
No, it's not debug. It's because transport has to start after workmanagercooordinator and this can't be done w/ <depends> to don't have circular deps